### ISSUE TITLE
NVDA freezing in Microsoft edge upon entering into an auto complete field

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#12187]
[nvda.log](https://github.com/nvaccess/nvda/files/6157062/nvda.log)

### Steps to reproduce:
1. Open Microsoft ege
2. Enter into an auto complete field by pressing ctrl+down arrow after NVDA says "autocomplete"
3. You will notice that NVDA will freeze 10-30 seconds after ctrl+down arrow is pressed
### Actual behavior:
NVDA will freeze after pressing ctrl+down arrow
### Expected behavior:
NVDA will not freeze and report the suggestions instead
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.4
#### Windows version:
windows 10 1909 and 2004 64-bit
#### Name and version of other software in use when reproducing the issue:
Microsoft edge 89.0.774.54 
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by Adriani90** [comment]
@fatihyuksek1 are you still having this issue with NVDA 2023.1 and Microsoft Edge 111? I cannot reprodue it on Windows 11.

**Comment by fatihyuksek1** [comment]
No, I'm not experiencing this issue anymore. It seems to be gone.

**Comment by Adriani90** [comment]
Thanks for reporting. Closing as works for me.
